main.go

```diff
@@ -219,6 +220,7 @@ func main() { // nolint:gocyclo
 		mgr,
 		namespace,
 		memberClientTimeout,
+		useClusterRL,
```

Comment:
just an idea - We could pass the `cluster.Tenant` string instead of using a bool. 

see my comment https://github.com/codeready-toolchain/toolchain-common/pull/359#discussion_r1496100784

Reply:
would be removing this